packages/xo-server/sample.config.toml

```diff
@@ -141,3 +141,22 @@ port = 80
 #
 # Default: '/var/lib/xo-server/data'
 #datadir = '/var/lib/xo-server/data'
+
+#=====================================================================
+
+# Configuration for remotes
+[remoteOptions]
+# Directory used to mount remotes
+#
+# Default: '/run/xo-server/mounts'
+#mountsDir = '/run/xo-server/mounts'
+
+# Timeout in milliseconds (set to 0 to disable)
+#
+# Default: '600e3'
+#timeout = 600e3
```

Comment:
This is an advanced option and it usually does not make sense for the users to change it, please remove it from this file :slightly_smiling_face: